packages/core/addon/serializers/table.js

```diff
@@ -5,14 +5,35 @@
 import VisualizationSerializer from 'navi-core/serializers/visualization';
 import { parseMetricName } from 'navi-data/utils/metric';
 import { get } from '@ember/object';
+import { isPresent, isNone } from '@ember/utils';
+import omit from 'lodash/omit';
 
-const TRANSFORMS = {
+const MAP_STRING_FIELD = {
   metric: column => Object.assign(column, { field: parseMetricName(column.field) }),
```

Comment:
Do these functions have the right column shape?

Reply:
the input may be either one of the two previous "versions":
`field: 'revenue(currency=USD)'` or 
`field: { metric: 'revenue', parameters: { currency: 'USD' } }`

Reply:
I was thinking of the output of each function. We're still outputting an object with `field` instead of `attributes: { name: ... `

Reply:
Yes it will be confusing, will change